frontend/src/components/ViewPasses/ViewPasses.js

```diff
@@ -5,7 +5,7 @@ import Pass from "./Pass"
 import { apiUrl } from "../../services/config"
 import { toast } from "react-toastify"
 
-const ViewPasses = () => {
+const ViewPasses = (props) => {
```

Comment:
Destructor props

Reply:
done

Reply:
only verify needed to be destructured in this one